README.md

```diff
@@ -1,17 +1,16 @@
 # angulartraining
```

Comment:
Please add to package.json the commmand, which will generate unit tests code coverage. Eg:
`"test:coverage": "ng test --source-map --code-coverage",`